Shared/Samples/Change viewpoint/README.metadata.json

```
@@ -0,0 +1,37 @@
+{
+    "category": "Maps",
+    "description": "Set the map view to a new viewpoint.",
+    "ignore": false,
+    "images": [
+        "change-viewpoint-1.png",
+        "change-viewpoint-2.png"
+    ],
+    "keywords": [
+        "animate",
+        "extent",
+        "pan",
+        "rotate",
+        "scale",
+        "view",
+        "zoom",
+        "Geometry",
+        "Map",
+        "MapView",
+        "Point",
+        "Viewpoint"
+    ],
+    "redirect_from": [
+        "/ios/latest/swift/sample-code/change-viewpoint.htm"
+    ],
```

```suggestion
    "redirect_from": [],
```
No need to redirect from iOS Sample Viewer.